python_modules/dagster/dagster/_core/definitions/auto_materialize_condition.py

```diff
@@ -45,6 +45,7 @@ class ParentMaterializedAutoMaterializeCondition(NamedTuple):
     """
 
     decision_type: AutoMaterializeDecisionType = AutoMaterializeDecisionType.MATERIALIZE
+    materialized_asset_keys: Optional[FrozenSet[AssetKey]] = None
```

Comment:
If we're being pedantic, this whole class should be called ParentUpdatedAutoMaterializeCondition, as an observable source asset upstream of this asset would also cause this condition to be applied.

It's easier to retroactively change the name of this class than change the name of the class and the field, so maybe we should call this `updated_asset_keys`.